docs/auth0_completion.md

```diff
@@ -3,50 +3,83 @@ layout: default
 ---
 # auth0 completion
 
-completion [bash|zsh|fish|powershell]
+## Loading completions
```

Comment:
Completely redesigned the docs for auth0 completion as they weren't rendering well before.